index.bs

```diff
@@ -5952,6 +5961,144 @@ However, [=authenticators=] that do not utilize [[!FIDO-CTAP]] do not necessaril
 :: [=largeblob|This extension=] directs the user-agent to cause the large blob to be stored on, or retrieved from, the authenticator. It thus does not specify any direct authenticator interaction for [=[RPS]=].
 
 
+## Device-bound public key extension (<dfn>devicePubKey</dfn>) ## {#sctn-device-publickey-extension}
+
+This [=client extension|client=] [=registration extension=] and [=authentication extension=] provides a [=[RP]=] with a "device continuity" signal. This is done by creating a [=[RP]=]-specific [=hardware-bound device key pair=] on the [=client device=], if such a key pair does not already exist for the [=user credential=] being created or exercised, and returning the attested [=device public key=] along with a signature by the [=device private key=] to the [=[RP]=]. This is done each time this [=devicePubKey=] extension is included with either a <code>{{CredentialsContainer/create()|navigator.credentials.create()}}</code> or <code>{{CredentialsContainer/create()|navigator.credentials.get()}}</code> call.
```

Comment:
The introduction describes the device public key as being created by the _client device_, but the authenticator extension processing steps describe it as being created by the _authenticator_. Those are probably synonyms in the case of platform authenticators, but the discrepancy is a bit weird to me.

Reply:
yes, good catch, this is an authenticator extension and that intro paragraph needs revision to reflect that.  I'll do that after we address your above uber-comment :)

Reply:
fixed in 0bb9aaa